geniza/corpus/forms.py

```diff
@@ -226,13 +226,15 @@ class DocumentSearchForm(forms.Form):
     }
 
     def filters_active(self):
+        # returns True if any filter fields are active in cleaned_data
```

Comment:
we should probably update the comment here (and should probably be a docstring, too)

```suggestion
        """Check if any filters are active; returns true if form fields other than sort or q are set"""
```